mise.toml

```diff
@@ -0,0 +1,16 @@
+[tools]
+
+# Core Dependencies
+go = "1.22.7"
+
+# Go Dependencies
+"go:github.com/golangci/golangci-lint/cmd/golangci-lint" = "1.64.5"
+"go:go.uber.org/mock/mockgen" = "0.5.0"
+"go:gotest.tools/gotestsum" = "1.12.0"
+"go:github.com/segmentio/golines" = "0.12.0"
+
+# Forge Dependencies
+cast = "v1.0.0"
```

Comment:
why do we need forge as a dependency?

Reply:
don't remember but had listed it in the issue ticket. I'm guessing some makefile target or some command in some script uses cast somewhere.